views/layouts/main.handlebars

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+		<link rel="stylesheet" href="/assets/css/style.css" type="text/css" />
```

Comment:
should we add:

<meta name="viewport" content="width=device-width, initial-scale=1">